tests/integration/tests/autocomplete.js

```diff
@@ -228,20 +228,22 @@ test('submit key', async () => {
   await page.keyboard.press('Enter');
   await page.waitForSelector('.autocomplete_suggestions', { hidden: true });
 
-  let center = await page.evaluate(() => {
-    return window.MAP_MOCK.getCenter();
-  });
+  let center = await page.evaluate(() =>
+    window.MAP_MOCK.getCenter()
+  );
 
   let firstFeatureCenter = mockAutocomplete.features[0].geometry.coordinates;
   expect(center).toEqual({ lat: firstFeatureCenter[1], lng: firstFeatureCenter[0] });
   await page.click('#clear_button');
 
   /* force specific query */
   responseHandler.addPreparedResponse(mockAutocompleteAllTypes, /autocomplete\?q=paris/);
-  await page.keyboard.type('paris');
+  await autocompleteHelper.typeAndWait('paris');
   await page.keyboard.press('Enter');
   await page.waitForSelector('.autocomplete_suggestions', { hidden: true });
 
+  await page.waitFor(300);
```

Comment:
Do you have idea why it's needed in the new implementation? Does it mean the React component is less reactive to user inputs?

Reply:
I have no idea, I don't understand this behavior meanwhile the first `expect` in this test pass.